nautobot/core/settings.json

```diff
@@ -0,0 +1,598 @@
+{
+    "type": "object",
+    "definitions": {
+        "callable": {
+            "type": "string"
+        },
+        "regex": {
+            "type": "string"
+        }
+    },
+    "properties": {
+        "ADMINS": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "default": []
+        },
+        "ALLOWED_HOSTS": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "default": []
+        },
+        "BANNER_BOTTOM": {
+            "type": "string",
+            "default": ""
+        },
+        "BANNER_LOGIN": {
+            "type": "string",
+            "default": ""
+        },
+        "BANNER_TOP": {
+            "type": "string",
+            "default": ""
+        },
+        "BRANDING_FILEPATHS": {
+            "type": "object",
+            "properties": {
+                "logo": {
+                    "type": "string",
+                    "default": ""
+                },
+                "favicon": {
+                    "type": "string",
+                    "default": ""
+                },
+                "icon_16": {
+                    "type": "string",
+                    "default": ""
+                },
+                "icon_32": {
+                    "type": "string",
+                    "default": ""
+                },
+                "icon_180": {
+                    "type": "string",
+                    "default": ""
+                },
+                "icon_192": {
+                    "type": "string",
+                    "default": ""
+                },
+                "icon_mask": {
+                    "type": "string",
+                    "default": ""
+                },
+                "header_bullet": {
+                    "type": "string",
+                    "default": ""
+                },
+                "nav_bullet": {
+                    "type": "string",
+                    "default": ""
+                }
+            }
+        },
+        "BRANDING_PREPENDED_FILENAME": {
+            "type": "string",
+            "default": "nautobot_"
+        },
+        "BRANDING_TITLE": {
+            "type": "string",
+            "default": "Nautobot"
+        },
+        "BRANDING_URLS": {
+            "type": "object",
+            "properties": {
+                "code": {
+                    "type": "string",
+                    "default": "https://github.com/nautobot/nautobot"
```

Comment:
Wonder if it'd be useful to define a `url` type for this property and similar ones? Actually, I think jsonschema may have a native type for this already?

Reply:
Looking through the doc [here](https://json-schema.org/understanding-json-schema/reference/type), I dont see a type for url. I think I will make a new type for that in this schema unless I am missing something

Reply:
https://json-schema.org/understanding-json-schema/reference/string#built-in-formats